### ISSUE TITLE
 Could you please provide the Training code?

### ISSUE THREAD

**Issue by yuanzhen2020** [crs4/Deep3DLayout#3]
Could you please provide the Training code?

**Comment by giannipint** [comment]
At the moment the pytorch implementation of data preparation and training is part of other non-academic projects and cannot be shared. However, our scripts basically follows those of meshrcnn , in particular for accumulating the loss of multiple stages (2 in our case, 3 for meshrcnn).
